src/components/App/App.js

```diff
@@ -8,21 +8,24 @@ export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: [],
+      movies: []
     }
   }
   componentDidMount = () => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
       .then(response => {
         if(!response.ok) {
           this.setState({allMoviesError: response.status});
+          return false;
         } else {
           return response.json();
         }
       }
     ).then(data => {
-      this.setState({movies: data.movies});
-    });
+      if (data) {
+        this.setState({movies: data.movies});
+      }
+    })
```

Comment:
I played around with having a catch but since we already have our error message implementation, the catch actually didn't change anything. if it feels better, down to add it though

Reply:
At this point it seems like whatever is working is "better"!